### ISSUE TITLE
Module '"mathjs"' has no exported member 'symbolicEqual'

### ISSUE THREAD

**Issue by ikcede** [josdejong/mathjs#3259]
# Bug

The latest NPM version of mathjs does not export the symbolicEqual function for Typescript.

# To Reproduce

```bash
# Install mathjs
npm install mathjs
```

```typescript
// page.ts: Try to import symbolicEqual
import {symbolicEqual} from 'mathjs'
```

The Typescript compiler will error

# To Fix

Adding symbolicEqual to the exported consts at the end of `types/index.d.ts` fixes the issue as then this code compiles:

```typescript
// page.ts
import {symbolicEqual} from 'mathjs'
console.log(symbolcEqual('1 + 2', '2 + 1') // true
```

**Comment by josdejong** [comment]
Thanks for reporting, I've implemented a fix (not yet published)

**Comment by josdejong** [comment]
The fix is published now in `v13.2.0`
